pyproject.toml

```diff
@@ -40,24 +40,28 @@ keywords = [
     "data science",
     "database",
 ]
-repository = "https://github.com/exasol/sqlalchemy-exasol"
-homepage = "https://www.exasol.com/"
-documentation = "https://exasol.github.io/sqlalchemy-exasol/"
-
 include = [
     "README.rst",
     "CHANGES.md",
     "LICENSE",
 ]
 exclude = []
 
+[tool.poetry.urls]
```

Comment:
is this new?

Reply:
@tkilias I don't think so, but just found out about it recently :D

![image](https://user-images.githubusercontent.com/600911/205018099-3dfe07d8-5a32-4a6c-bde7-665f9318ffd5.png)